handbook/handbook.sop.todos.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 0292b34e-47eb-4499-8f49-d9891accdb3d
+title: Todos
+desc: ''
+updated: 1623169745695
+created: 1622146649559
+---
+
+## Summary
+
+This goes over how we format todo notes at Dendron.
+
+## Sections
+- high: something that should be completed today
+- mid: something that can be postponed
+- new: something that came up and needs to be prioritized
+
+## Status
+
+This is added to the end of a todo. 
+
+
+- p: post poned (move to next day)
```

Comment:
Yeah, this is helpful, I haven't been using ~ 

nit: postponed.

Reply:
will change